### ISSUE TITLE
Tamed Dragons appear invulnerable

### ISSUE THREAD

**Issue by OatmealDefender** [adam9899/Issues#235]
I cannot seem to damage tamed dragons.
To replicate:
Give yourself a fire dragon egg (I used red).  Hatch said egg so its tame. Feed dragon dragon meal until its day 120ish so its t5 (120 or so dragon meal).  Put the dragon in a dragon horn so you can summon/desummon it at whim.
Summon the dragon, start hitting it.

Behavior:
Dragon will take hit if you hit it in correct place (mid body) and make an injured sound, roar, and appear to suffer side effects of the hit.  (as I was using a sword with the ice dragon blood modification, this was clear as it was placed in an ice ccube).  Dragon's health will not move in the gui.  To make sure it wasn't just a visual glitch, I hit the dragon several hundred times, which with the sword in question would have killed a wild t5 many times over.  I also tried having someone else on the server hit the dragon, no dice.

TLDR: Tamed dragons (at least tamed t5 fire dragons, as that's all I had to test with) appear invulnerable.

As to why I wanted to kill my own tamed dragon, I was going to breed them and kill excess ones for blood so I could make dragonsteel without wastefully genning a ton of terrain to get enough wild ones to kill.

**Comment by OatmealDefender** [comment]
Quick comment: Version 1.3.2

**Comment by OatmealDefender** [comment]
By way of update...it appears to be a team issue, as mobs can still damage the dragon, albeit barely.  I will close the issue to not waste your time.
